packages/react-chat/src/contexts/RuntimeContext/useRuntimeState.ts

```diff
@@ -127,6 +135,8 @@ export const useRuntimeState = ({ assistant, config }: Settings) => {
     setOpen(false);
   };
 
+  const getTurns = () => sessionRef.current.turns;
```

Comment:
we could consider expanding this to `getState()` to get the latest state (including `isOpen` and `indicator`)